api-changes.md

```diff
@@ -3,6 +3,120 @@
 Informal notes ahead of the release. This document will be merged
 into HISTORY when the release is cut.
 
+## AST changes
+
 - Remove non-device selectors.
 
+- Table, Stream, Actions are no longer used. Instead, everything is
+  an Expression, which could be a table Expression (Index, Sort, Aggregation),
+  a table-or-stream Expression (Filter, Projection), or any of the
+  three types (FunctionCall, Invocation)
```

Comment:
what three types? what is the difference between `FunctionCall` and `Invocation`?